packages/aragon-wrapper/src/index.js

```diff
@@ -791,6 +777,35 @@ export default class Aragon {
         throw new Error(`No method named ${methodName} on ${destination}`)
       }
 
+      const methodABI = app.abi.find(
+        (method) => method.name === methodName
+      )
+      if (!methodABI) {
+        throw new Error(`${methodName} not found on ABI for ${destination}`)
+      }
+
+      let transactionOptions = {
```

Comment:
This declaration needs to be moved to outside the `if` scope, since `createForwarderTransaction()` needs it later